builtin/nomad/jobspec/platform.go

```diff
@@ -83,6 +83,9 @@ func (p *Platform) Deploy(
 	// Set our deployment ID on the meta.
 	job.SetMeta(metaId, result.Id)
 
+	// Update our client to use the Namespace set in the jobspec
+	client.SetNamespace(*job.Namespace)
```

Comment:
Does this need to be a pointer here, or can we use the `job.Namepsace` string value directly? 🤔

Reply:
We checked and it is a pointer already, @briancain was mistaken.